src/main/java/uk/gov/hmcts/reform/blobrouter/services/report/ReportService.java

```diff
@@ -1,24 +1,34 @@
 package uk.gov.hmcts.reform.blobrouter.services.report;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.blobrouter.config.ServiceConfiguration;
 import uk.gov.hmcts.reform.blobrouter.data.reports.ReportRepository;
 import uk.gov.hmcts.reform.blobrouter.model.out.EnvelopeSummaryItem;
+import uk.gov.hmcts.reform.blobrouter.model.out.reports.EnvelopeCountSummaryReportItem;
 
 import java.time.Instant;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 import java.util.List;
+import java.util.Objects;
 
 import static java.util.stream.Collectors.toList;
 import static uk.gov.hmcts.reform.blobrouter.util.TimeZones.EUROPE_LONDON_ZONE_ID;
 
 @Service
 public class ReportService {
     private final ReportRepository reportRepository;
+    public static final String TEST_CONTAINER = "bulkscan";
```

Comment:
we do no need filtering.

Reply:
If we follow how it is done in processor we should use filtering. And I think this is right.

Reply:
bulkscanauto is missing.
and we use these containers in lower environments , if we always filter result will be wrong in lower environments.

Reply:
This can be private. And is it used anywhere?